html/styles.css.php

```diff
@@ -84,26 +84,57 @@
     width:25px;
     padding-left:20px;
 }
-#date, #time, .label, .wind_now {
+
+.label, .wind_now {
     font-size:<?= $YANPIWS['font_time_date_wind']?>pt;
     font-weight: bold;
 }
 
+.big_time {
+    font-size: 127pt;
+    text-align: center;
+    width: 100%;
+}
+
+.small_time {
+    font-size: <?= $YANPIWS['font_time_date_wind']?>pt;
+    text-align: left;
+    width: fit-content;
+    padding: 5px;
+    display: inline-block;
+}
+
+.small_time_parent {
+    width: 100%;
+}
+
+.big_clock_hide {
+    display: block;
+}
+
```

Comment:
These look fine (i'll test after you finalize the PR), but be sure you test on a 5" 800x480 viewport for the target screen in the [hardware section](https://github.com/mrjones-plip/YANPIWS#hardware). [Chrome](https://developers.google.com/web/tools/chrome-devtools/device-mode/#viewport) and [FF](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode) got yer back!

(Super extra bonus points if you test in the 3.5" 480x320 too, but I haven't tested this in a while - I suspect it won't work ;)

Reply:
Did you get a chance to test this in the 800x480 viewport in your browser with dev tools?